api/controllers/UserController.js

```diff
@@ -268,12 +268,12 @@ module.exports = {
         promises.push(Ban.updateAutomod(req.user.redToken, req.params.username, 'SVExchange', unique_fcs));
         promises.push(Ban.addUsernote(req.user.redToken, req.user.name, 'pokemontrades', req.params.username, req.params.banNote));
         promises.push(Ban.addUsernote(req.user.redToken, req.user.name, 'SVExchange', req.params.username, req.params.banNote));
-        promises.push(Ban.removeTSVThreads(req.user.redToken, req.params.username));
+        promises.push(Ban.markTSVThreads(req.user.redToken, req.params.username));
         promises.push(Ban.updateBanlist(req.user.redToken, req.params.username, req.params.banlistEntry, unique_fcs, igns, req.params.knownAlt));
         promises.push(Ban.localBanUser(req.params.username));
       }
       Promise.all(promises).then(function () {
-        console.log('Process to ban /u/' + req.params.username + 'was completed successfully.');
+        console.log('Process to ban /u/' + req.params.username + ' was completed successfully.');
```

Comment:
The completion message was added in a [2.3 refactor](https://github.com/YaManicKill/flairhq/blame/version-2.3/api/controllers/UserController.js#L276) -- it doesn't exist in 2.2 at all.